privacy_estimates/binomial_proportion.py

```diff
@@ -77,6 +104,41 @@ def compute_eps_lo_hi(count: AttackResults, delta: float, alpha: float, method:
     return eps_lo, eps_hi
 
 
+def compute_delta_lo_hi(count: AttackResults, epsilon: float, alpha: float, method: str) -> Tuple[float, float]:
+    """
+    Computes a confidence interval for delta from equal-tailed confidence intervals for the false positive and
+    false negative rates of a membership inference attack, based on 100*(1 - alpha)% confidence intervals for epsilon
+
+    Args:
+        count (AttackResults):
+            Number of false positives, false negatives, true positives, and true negatives
+        epsilon (float):
+            Differential privacy :math:`\epsilon` parameter
+        alpha (float):
+            Significance level
+        method ({'beta', 'jeffreys'}):
+            Method to use for confidence intervals.
+            We use central (aka equal-tailed) methods available from `statsmodels`.
+            - `beta` : Clopper-Pearson interval based on Beta distribution
+            - `jeffreys` : Jeffreys Bayesian Interval
+            Default is 'jeffreys'
+
+    Returns:
+        delta_low, delta_hi (Tuple[float, float]): two-sided interval [delta_lo, delta_hi]
+    """
+    fnr_l, fnr_r, fpr_l, fpr_r = compute_binomial_proportion(count, alpha, method)
+
+
+    # Compute delta from top side of the confidence-square for smaller delta width
+    delta_up = delta_from_fnr_fpr(fnr=fnr_r, fpr=fpr_r, epsilon=epsilon)
+    delta_low = delta_from_fnr_fpr(fnr=fnr_l, fpr=fpr_l, epsilon=epsilon)
```

Comment:
Shouldn't these be flipped? In the top-left quadrant we have

$$ \delta = \max (0, 1 - \textrm{FPR} - e^\varepsilon\ \textrm{FNR}) $$

So the lower endpoint is reached at the upper endpoints of the intervals for $\textrm{FNR}$ and $\textrm{FPR}$.

Reply:
Sorry, yes you are right.. the up and low correspond to $1 - \delta$ and not $delta$. For delta they can be flipped.